android/fastlane/Fastfile

```diff
@@ -71,15 +77,15 @@ lane :android_build_beta do
 
   versionName = "Inji #{git_tag}"
 
-  gradle(task: "clean bundleBetaRelease")
+  gradle(task: "clean bundle#{INJI_FLAVOR}Release")
   upload_to_play_store(
   track: 'alpha',
   release_status: 'completed',
   version_name: versionName,
   )
 
   slack(
-      message: "Inji [Beta]- #{versionName} is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.", 
+      message: "Inji - #{INJI_FLAVOR} Wallet- #{versionName} is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.", 
```

Comment:
Can we use app_name created for each flavor in build gradle

Reply:
Have used a function to get the app name.